### PR TITLE
coala_settings: Remove "Documentation" from title

### PR DESCRIPTION
**Pull request by cociorbaandrei**
Closes https://github.com/coala/coala/issues/3319

### Checklist

- [ ] I have rebased properly. Please see [our tutorial on rebasing](http://coala.io/git#rebasing).
- [x] I have gone through the [commit guidelines](http://coala.io/commit) and I've followed them.
- [x] I have followed the [guidelines for the commit shortlog](http://coala.io/commit#shortlog).
- [x] I have followed the [guidelines for the commit body](http://coala.io/commit#commit-body).
- [x] I have included the issue URL with the 'Fixes' keyword if the commit fixes a *real bug* and the 'Closes' keyword if it adds a feature or enhancement. For details, check the [issue reference section in our commit guidelines](http://coala.io/commit#issue-reference).
- [x] I have [run all the tests](http://api.coala.io/en/latest/Developers/Executing_Tests.html) and they all pass. If any CI (below) is not green, please fix them. If GitMate issues appear you will have to amend your commits, pushing new commits on top is not sufficient!

### CODE REVIEW

File: docs/Developers/coala_settings.rst [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Documentation coala settings
+ Coala settings
```
Comment (andreidinu) [inline docs/Developers/coala_settings.rst]: coala is always written with a lowercase c.

File: docs/Developers/coala_settings.rst [hunk 2]
```diff
@@ -1,4 +1,4 @@
...
 ============================
```
Comment (sils) [inline docs/Developers/coala_settings.rst]: underline length doesn't match title

File: docs/Developers/coala_settings.rst (hunk 1 above)
Comment (sils) [inline docs/Developers/coala_settings.rst]: also why's there a space before?

**Comment by meetmangukiya** [comment]
ack 8c0f72f

**Comment by Mixih** [comment]
unack  	8c0f72f

**Comment by Mixih** [comment]
unack 8c0f72f

**Comment by Mixih** [comment]
@cociorbaandrei you need to set authorship in your git. Please do `git config --global user.email "<your github email>"` and amend you commit using `git commit -a --amend`. After that, force push to your fork and it should fix itself. Also, you will probably need to rebase by then. WHATEVER YOU DO, DON'T PUSH THE GITHUB BUTTON!!! Instead, follow http://api.coala.io/en/latest/Developers/Git_Basics.html#rebasing

**Comment by Mixih** [comment]
hmmm your authorship still isn't right. Are you sure you git email matches your github one? Also, try the config without --global. Make sure there are no typos

**Comment by ashwini0529** [comment]
@cociorbaandrei please rebase.

**Comment by ashwini0529** [comment]
This PR needs a rebase @cociorbaandrei

**Comment by sils** [comment]
@Mixih please avoid writing stuff in upper case letters directed to individual people, that's usually a good ground rule to avoid people being offended although I know you don't mean it like that a newcomer doesn't necessarily.

@cociorbaandrei your commit seems good, if you want we can merge it like this *but* the authorship info is still wrong which means if we merge it like that it won't appear on your github profile. Also this needs a rebase, but if you don't plan to stick with us we can surely do that for you so you have your contribution merged. Let us know how you want to proceed! In any case, if you didn't already, consider [giving us some feedback](http://coala.io/newform).

https://github.com/coala/coala/pull/3320/commits/0b40f9ee80bdb2e0c2f7d8ecacee5748acced48b is ready

**Comment by Mixih** [comment]
@cociorbaandrei Hello there! Just checking in on you to see if you still want to work on this. There is only a little bit you still need to do for it to get merged. Would you be willing to come back and give it a little love?

**Comment by Makman2** [comment]
Merging without authorship information -> Cherry-picked into https://github.com/coala/coala/pull/3520

**Pull request closed without merging**